packages/ssi-service/src/feedback-panel/api.js

```diff
@@ -1,67 +1,56 @@
-    }
-
-    if ( this.authToken ) {
-      headers.Authorization = 'Bearer ' + this.authToken;
-    } else if ( window.OpAuthHelper?.jwtToken ) {
-      headers.Authorization = 'Bearer ' + window.OpAuthHelper.jwtToken;
-    }
-
-    return headers;
-  }
-
-  /**
-   * Use a custom endpoint or auth token for the Feedback Graphql APIs
-   * @param {{ endpoint: string, authToken: string }} basePath
-   */
-  config ( { endpoint, authToken } ) {
-    if ( endpoint ) {
-      this._apiBasePath = endpoint;
-    }
-    if ( authToken ) {
-      this.authToken = authToken;
-    }
-  }
-
-  /**
-   * Sends a graphql request
-   */
-  request ( query, variables = undefined ) {
-    return fetch( this._apiBasePath, {
-      method: 'POST',
-      headers: this._headers,
-      body: JSON.stringify( {
-        query,
-        variables,
-      } ),
-    } )
-      .then( res => {
-        if ( !res ) {
-          throw new Error( 'The Server returned an empty response' );
-        }
-        return res.json();
-      } )
-      .then( res => {
-        if ( res.errors ) {
-          throw new Error( 'There were some errors in the query' + JSON.stringify( res.errors ) );
+sendFeedback = (feedbackInput) => {
+    let body = JSON.stringify({
+        query: `mutation CreateFeedback($input: FeedbackInput!) {
+        createFeedback(input: $input) {
+            _id
+            ticketUrl
+            }
+        }`,
+        variables: {
+            "input": feedbackInput
         }
-        return res.data;
-      } );
-  }
+    });
+    return fetch(process.env.APPS_BASE_API, {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json',
+            'Authorization': `${(window.OpAuthHelper.jwtToken) ? `Bearer ${window.OpAuthHelper.jwtToken}` : null}`
+        },
+        body: body
+    })
+        .then(res => {
+            if (!res) {
+                (window.OpNotification) ? window.OpNotification.danger({ subject: `Error in Feedback Submission`, body: `The Server returned an empty response.` }) : alert('Error in Feedback Submission');
+                throw new Error('The Server returned an empty response.');
+            }
+            return res.json();
+        })
+        .then(res => {
+            if (res.errors) {
+                (window.OpNotification) ? window.OpNotification.danger({ subject: `Error in Feedback Submission` }) : alert('Error in Feedback Submission');
+                throw new Error('There were some errors in the query' + JSON.stringify(res.errors));
+            } else if (!res.errors) {
+                (window.OpNotification) ? window.OpNotification.success({ subject: `Submitted Feedback`, link: response?.data?.createFeedback?.ticketUrl || null }) : alert('Submitted Feedback');
+            }
+            return res.data;
+        });
 }
 
-const APIServiceInstance = new APIService();
-export default APIServiceInstance;
+module.exports.sendFeedback = sendFeedback;
```

Comment:
Newline at the end of a file.
```suggestion
module.exports.sendFeedback = sendFeedback;

```